### ISSUE TITLE
Key error in carla_OD_train.tar.gz

### ISSUE THREAD

**Issue by mzweilin** [twosixlabs/armory#1216]
Dear Armory team,

I have downloaded the training data set [`carla_OD_train.tar.gz`](https://armory-public-data.s3.us-east-2.amazonaws.com/carla/carla_OD_train.tar.gz) for the Carla object detection model, but I found that the annotation key `area` is written as `areas` (plural form) in the json file.

Though the script [`carla_obj_det_train.py`](https://github.com/twosixlabs/armory/blob/r0.14.0/armory/data/carla_object_detection/carla_obj_det_train.py#L143) you provide makes an inline conversion for this, could you please fix the key in json so that it's easier for us to train models with other tools?

https://github.com/twosixlabs/armory/blob/r0.14.0/armory/data/carla_object_detection/carla_obj_det_train.py#L143

BTW, could you also kindly provide the validation dataset in the same COCO format other than `TFRecords`?

**Comment by Lodour** [comment]
FYI, the validation set is available here: https://github.com/twosixlabs/armory/blob/r0.14.0/armory/data/adversarial/carla_obj_det_dev.py#L29

Typo can be fixed by `sed -i 's/areas/area/' coco_annotations.json`

**Comment by mzweilin** [comment]
Thanks, @Lodour
